packages/file_selector/file_selector/example/lib/get_multiple_directories_page.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:file_selector/file_selector.dart';
+import 'package:flutter/material.dart';
+
+/// Screen that allows the user to select one or more directories using `getDirectoryPaths`,
+/// then displays the selected directories in a dialog.
+class GetMultipleDirectoriesPage extends StatelessWidget {
+  /// Default Constructor
```

Comment:
We can probably kill this line. It doesn't add anything. If for some reason we need it, it should end in a period.

Reply:
It's because we have `public_member_api_docs` on, so you have to either put a basically useless comment, or an override. I'll replace this with my go-to not very helpful comment for example constructors that it returns a new instance of the page, since at least that's explaining what it does rather than what it is.